protostar/commands/test/test_command.py

```diff
@@ -182,6 +184,7 @@ async def run(self, args: Namespace) -> TestingSummary:
     async def test(
         self,
         targets: List[str],
+        pure_cairo_vm: bool = False,
```

Comment:
I would name it `use_cairo_test_runner`, as we named it `CairoCompiler` and `StarknetCompiler`

Reply:
done